oak-security-spi/src/main/java/org/apache/jackrabbit/oak/spi/security/user/UserConstants.java

```diff
@@ -265,4 +265,12 @@ public interface UserConstants {
      * @since Oak 1.3.3
      */
     int PASSWORD_HISTORY_DISABLED_SIZE = 0;
+
+    /**
+     * Optional configuration parameter indicating if the anonymous user can be disabled or not.
+     * By default, the anonymous user can be disabled.
+     *
+     * @since Oak 1.73.0
```

Comment:
there is no Oak 1.73.0; Oak uses the odd-even versioning scheme, so there is an official 1.70.0, 1.72.0, 1.74.0 etc.

Reply:
Ok, I wasn't aware of this. I took it based from the pom.xml. Will change it to 1.74 then .